scripts/deploy-ebpf.ps1.in

```diff
@@ -283,7 +284,7 @@ OPTIONS:
 }
 
 $build_directory="$PSScriptRoot"
-if ($vm -eq $null) {
+if ($null -eq $vm) {
```

Comment:
Why this change?  Seems unnecessary for this PR

Reply:
Best practice in PS scipts.

Reply:
Please provide a reference for that claim, since I don't think it's helpful.

Reply:
agree as discussed, changing this also in other scripts.